src/feature/relay/dns.c

```diff
@@ -1357,6 +1357,36 @@ evdns_err_is_transient(int err)
   }
 }
 
+/**
+ * Return number of configured nameservers in <b>the_evdns_base</b>.
+ */
+size_t
+number_of_configured_nameservers(void)
+{
+  return evdns_base_count_nameservers(the_evdns_base);
+}
+
+/**
+ * Return address of configured nameserver in <b>the_evdns_base</b>
+ * at index <b>idx</b>.
+ */
+tor_addr_t *
+configured_nameserver_address(const size_t idx)
+{
+ struct sockaddr_storage sa;
+ ev_socklen_t sa_len = sizeof(sa);
+
+ if (evdns_base_get_nameserver_addr(the_evdns_base, (int)idx,
```

Comment:
This function wasn't added to Libevent until 2.1.6-beta; We only require 2.0.x, I think.  If you want to use it, we need to check for its presence in autoconf, and only call it when it's available.

Reply:
Oh wait, I see that you added a check later.  Great.